src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -121,6 +121,32 @@ public void remove(ReadOnlyPerson toRemove) throws PersonNotFoundException {
             throw new PersonNotFoundException();
         }
     }
+    /**
+     * Sorts all persons in list by name.
+     */
+    public void sort() {
+        ArrayList<String> fullNames = new ArrayList<>();
```

Comment:
It's good that you implemented your own sorting algorithm. Do you think it might be possible to use the built-in Java List.sort() command with a lambda expression?